formats/base_links.json

```diff
@@ -18,6 +18,9 @@
     "organisations": {
       "$ref": "#/definitions/guid_list"
     },
+    "lead_organisations": {
```

Comment:
Could you add a "description" field to this object, and also to the "organisations" one above?  These descriptions are just documentation for us, but are very useful to ensure that link types have a consistent meaning.

In particular, the `organisations` description should mention that this is all organisations associated with the content, duplicating the entries which are put in the `lead_organisations` field.  And the `lead_organisations` description should mention that all entries added to this field must also be added to the "organisations" field.